src/9-v04x-migration/index.md

```diff
@@ -0,0 +1,92 @@
+---
+date: 2016-02-12
+id: 9-v04x-migration
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "9-v04x-migration", link: "./" }
+tags: gateway, bootstrap, infrastructure
+title: Migrating ipfs.io from go-ipfs 0.3.x to 0.4.0
+author: Lars Gierth
+collection: posts
+---
+
+Good news everyone! We are about to release go-ipfs 0.4.0,
+which contains lots of great changes and enhancements.
+
+There are breaking changes
+which prevent 0.4.x nodes from communicating with 0.3.x nodes.
+This means that there will be, and already are, two separate networks.
+We'll call these networks v03x and v04x.
+IPFS nodes built from the master branch are already part of v04x.
+The Docker image `ipfs/go-ipfs` is built from master and thus also part of v04x.
```

Comment:
we should have the docker image follow the tags i have in `jbenet/go-ipfs`:
- `hub.docker.com/u/ipfs/go-ipfs@release` follows `github.com/ipfs/go-ipfs@release` (0.3.11 now)
- `hub.docker.com/u/ipfs/go-ipfs@latest` follows `github.com/ipfs/go-ipfs@release`  (0.3.11 now)
- `hub.docker.com/u/ipfs/go-ipfs@master` follows `github.com/ipfs/go-ipfs@master` (0.4.0 now)